### PR TITLE
Remove dead third-party clients

### PR DESCRIPTION
**Pull request by intrnl**
ref: https://github.com/bluesky-social/atproto-website/issues/203

This pull request removes third-party clients that are now dead, whether it's because of the repository being archived, or the site simply no longer being reachable.

### CODE REVIEW

File: content/community/projects.md [hunk 1]
```diff
@@ -78,11 +78,9 @@ The official Bluesky app is available on the [iOS App](https://apps.apple.com/us
 - [Klearsky](https://klearsky.pages.dev/) by [@mimonelu.net](https://bsky.app/profile/did:plc:ilxxgyz7oz7mysber4omeqrg)
 - [Laika](https://laika-bluesky.web.app/) by [@makoto.bsky.social](https://bsky.app/profile/did:plc:tr5cijtwcpl2dqrtmjsmmcow)
 - [Skylight](https://penpenpng.github.io/skylight) by [@penpenpng.bsky.social](https://bsky.app/profile/did:plc:ahj2yuo63gaxyhbgn5ai6jt2)
-- [The Blue](https://the-blue.shino3.net) by [@shino3.bsky.social](https://bsky.app/profile/did:plc:rpwpuzu2yyiuufm3232d7pm5)
```
Comment (emilyliu7321) [inline content/community/projects.md:81]: why is this one removed? seems to still be functional to me

Comment (intrnl) [reply content/community/projects.md:81]: Screenshot_2023-09-23-05-26-17-45_984e1414ae90666a90f12ff17ec14a7f

I'm not sure if this is indicative of being usable, it's a farewell message.

Comment (emilyliu7321) [reply content/community/projects.md:81]: ah thanks for the screenshot! the client loads a page that allows me to login on first click (screenshot below), but switches to the "see you again" screen on refresh. going to optimistically attribute this to a bug for now, but we can revisit this link after some time to see if it's still functional 

Screen Shot 2023-09-22 at 4 07 26 PM

mind putting this line back? then i'll approve the pr 👍  thanks for reviewing all of the clients! 🙏

Comment (intrnl) [reply content/community/projects.md:81]: very likely that there might be some service worker caching going and the new "farewell page" replacing it didn't replace the SW as well, causing the stale cache to be loaded instead

I'll come around to this PR in a an hour or two

Comment (intrnl) [reply content/community/projects.md]: should be good to go

**Review by emilyliu7321 — APPROVED** [review]
thank you!

**Pull request merged**